Training
Epoch 1/1 [Train]:   2%|█▏                                                                            | 3/196 [00:02<02:55,  1.10it/s, Loss=0.0260]
Traceback (most recent call last):
  File "/home/vatsal/NWM/fno_sevir/fno_sevir.py", line 96, in <module>
    avg_train_loss.append(loss.item())
                          ~~~~~~~~~^^
KeyboardInterrupt
Exception ignored in atexit callback <function _start_and_connect_service.<locals>.teardown_atexit at 0x7a370cc38e00>:
Traceback (most recent call last):
  File "/home/vatsal/NWM/SSBware/senv/lib/python3.13/site-packages/wandb/sdk/lib/service_connection.py", line 94, in teardown_atexit
    conn.teardown(hooks.exit_code)
  File "/home/vatsal/NWM/SSBware/senv/lib/python3.13/site-packages/wandb/sdk/lib/service_connection.py", line 226, in teardown
    self._router.join()
  File "/home/vatsal/NWM/SSBware/senv/lib/python3.13/site-packages/wandb/sdk/interface/router.py", line 75, in join
    self._thread.join()
  File "/home/vatsal/miniconda3/lib/python3.13/threading.py", line 1092, in join
    self._handle.join(timeout)
KeyboardInterrupt:
